src/test/java/uk/ac/ebi/subs/eva/EvaVcfSubmission.java

```diff
@@ -55,20 +53,20 @@ public void B_addSample() throws Exception {
         addSample(submissionUrl, sampleAlias, token, pm);
     }
 
-    @Test
-    public void C_uploadFile() throws Exception {
-        UploadUtils.uploadFile(token, submissionUrl, fileName);
-        TestUtils.waitForFileValidationCompletion(token, submissionUrl);
-    }
+//    @Test
+//    public void C_uploadFile() throws Exception {
+//        UploadUtils.uploadFile(token, submissionUrl, fileName);
+//        TestUtils.waitForFileValidationCompletion(token, submissionUrl);
+//    }
 
-    @Test
-    public void D_addAnalysis() throws Exception {
-        String analysisJson = TestJsonUtils.getSeqVarAnalysisJson(analysisAlias, studyAlias, sampleAlias, fileName, fileType);
-        String analysisUrl = TestUtils.createSubmittable(token, "variantCalls", submissionUrl, analysisJson);
-        TestUtils.waitForValidationResults(token, analysisUrl);
-        ValidationResult validationResult = TestUtils.getValidationResultForSubmittable(analysisUrl, token);
-        assertNoErrorsInValidationResult(validationResult);
-    }
+//    @Test
+//    public void D_addAnalysis() throws Exception {
+//        String analysisJson = TestJsonUtils.getSeqVarAnalysisJson(analysisAlias, studyAlias, sampleAlias, fileName, fileType);
+//        String analysisUrl = TestUtils.createSubmittable(token, "variantCalls", submissionUrl, analysisJson);
+//        TestUtils.waitForValidationResults(token, analysisUrl);
+//        ValidationResult validationResult = TestUtils.getValidationResultForSubmittable(analysisUrl, token);
+//        assertNoErrorsInValidationResult(validationResult);
+//    }
```

Comment:
Can we put a reason next to commented out code please. Its easy to forget such things after a short amount of time.

Reply:
Excellent suggestion. Thanks. I fixed it